doc/python.rst

```diff
@@ -759,9 +759,11 @@ or
 
 .. code-block:: python
 
-  # pool shutdown on object deletion, interpreter shutdown, or call to rollout_.shutdown_pool
+  # pool shutdown on object deletion or call to rollout_.shutdown_pool
+  # to ensure clean shutdown of threads, call shutdown_pool before interpreter exit
   rollout_ = rollout.Rollout(nthread)
```

Comment:
Please use a keyword argument to specify nthread. I can imagine we'll want to add more at some point.